roles/neutron-data-network/tasks/main.yml

```diff
@@ -59,6 +59,45 @@
     neutron-metadata-agent:
       config_files: /etc/neutron/metadata_agent.ini
 
+- name: neutron lbaas config
+  template:
+    src=etc/neutron/neutron_lbaas.conf
+    dest=/etc/neutron/neutron_lbaas.conf
+    mode=0644
+  notify:
+    - restart neutron services
```

Comment:
Additionally, I don't see the lbaas service being added to the list of services in the `restart neutron services` handler.